plugin-desktopswitch/desktopswitch.cpp

```diff
@@ -278,13 +292,11 @@ void DesktopSwitch::realign()
     {
         mLayout->setRowCount(mShowOnlyActive ? 1 : mRows);
         mLayout->setColumnCount(0);
-        mDesktops->setDesktopLayout(NET::OrientationHorizontal, columns, mRows, mWidget.isRightToLeft() ? NET::DesktopLayoutCornerTopRight : NET::DesktopLayoutCornerTopLeft);
```

Comment:
The `columns` computation is not needed here any more...
```
oss/lxqt/lxqt-panel/plugin-desktopswitch/desktopswitch.cpp: In member function ‘virtual void DesktopSwitch::realign()’:
oss/lxqt/lxqt-panel/plugin-desktopswitch/desktopswitch.cpp:289:9: error: unused variable ‘columns’ [-Werror=unused-variable]
  289 |     int columns = static_cast<int>(ceil(static_cast<float>(m_desktopCount) / mRows));
      |         ^~~~~~~
cc1plus: all warnings being treated as errors
```

Reply:
> remove the unused variable in realign()

Oh, didn't see it. Thanks!

Reply:
Done and squashed.

Reply:
> note that I don't see the CPU load with current master

Yes, your distro is Debian, if I correctly remember. It'll time a while until the issue comes to Debian.

However, changing desktop layout inside `realign()` was a potential danger.

Thanks for the fast review!